### PR TITLE
feat: add v3 to bol template

### PR DESCRIPTION
**Pull request by ChenRen93**

### CODE REVIEW

File: src/templates/BillOfLading/sampleV3.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { v3 } from "@govtechsg/open-attestation";
+import { BillOfLadingSchemaV3 } from "./types";
+
+export const BillOfLadingSampleV3: BillOfLadingSchemaV3 = {
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "http://localhost:3000/BOL",
+    "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json",
```
Comment (simboonlong) [inline src/templates/BillOfLading/sampleV3.ts:8]:
```suggestion
    "https://www.w3.org/2018/credentials/v1",
    "https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json",
    "https://schemata.openattestation.com/io/tradetrust/bill-of-lading/1.0/bill-of-lading-data.json",
```

**Comment by simboonlong** [comment]
able to help update the readme table too? create 2 new columns, v2, v3, then indicate which template has which support

File: src/templates/BillOfLading/sampleV3.ts (hunk 1 above)
Comment (ChenRen93) [reply src/templates/BillOfLading/sampleV3.ts]: done, changed it to https://schemata.openattestation.com/io/tradetrust/bill-of-lading/1.0/bill-of-lading-context.json

**Review by simboonlong — APPROVED** [review]
🎉

**Pull request merged**